v5/core/model/analytics.go

```diff
@@ -0,0 +1,30 @@
+package model
+
+import "time"
+
+type Analytics interface {
+	CaptureImpressions([]Impression)
+	InitiateIntervalReporting(interval time.Duration)
+	IsAnalyticsReportingDisabled() bool
+}
+
+type Impression struct {
+	Timestamp         float64     `json:"time"`
+	FlagName          string      `json:"flag"`
+	ExperimentId      string      `json:"experimentId,omitempty"`
+	ExperimentVersion string      `json:"experimentVersion,omitempty"`
+	Value             interface{} `json:"value"`
+	Type              string      `json:"type"`
+	Count             float64     `json:"count,omitempty"`
+}
+
+type SDKEventBatch struct {
+	AnalyticsVersion string       `json:"analyticsVersion"`
+	SDKVersion       string       `json:"sdkVersion"`
+	Timestamp        float64      `json:"time"`
```

Comment:
We don't. But I want to keep it in case later we want or need to consilidate timestamp counts.
I thought about this for a while. You aren't the first to notice ;)